src/components/App.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from "react";
+import {
+  createDataContext,
+  createItems,
+  createNewCollection,
+  createTable,
+  getAllItems,
+  getDataContext,
+  initializePlugin,
+  addComponentListener,
+} from "@concord-consortium/codap-plugin-api";
+import "./App.css";
+
+const kPluginName = "Sample Plugin";
+const kVersion = "0.0.1";
+const kInitialDimensions = {
+  width: 380,
+  height: 680
+};
+const kDataContextName = "SamplePluginData";
+
+export const App = () => {
+  const [codapResponse, setCodapResponse] = useState<any>(undefined);
+  const [listenerNotification, setListenerNotification] = useState<string>();
+  const [dataContext, setDataContext] = useState<any>(null);
+
+  useEffect(() => {
+    initializePlugin({pluginName: kPluginName, version: kVersion, dimensions: kInitialDimensions});
+
+    // this is an example of how to add a notification listener to a CODAP component
+    // for more information on listeners and notifications, see
+    // https://github.com/concord-consortium/codap/wiki/CODAP-Data-Interactive-Plugin-API#documentchangenotice
+    const createTableListener = (listenerRes: any) => {
```

Comment:
Since this is a starter plugin it would be best not to use `any` here to help with future type checking.  I looked in the api and `listenerRes` should be of type `ClientHandler`.  You could also just inline this with the `addComponentListener` and not have to import the type, like: `addComponentListener((listenerRes) => {...})`.